Packs/AzureSentinel/Integrations/AzureSentinel/AzureSentinel.py

```diff
@@ -135,15 +136,77 @@ def incident_data_to_demisto_format(inc_data):
     return formatted_data
 
 
-def get_update_incident_request_data(client: Client, args: Dict[str, str]):
+def watchlist_data_to_xsoar_format(watchlist_data):
```

Comment:
Please add docstrings here and in every function or method that doesn't have one.